File: src/Philips.IBE.Service.WebAgent/philips.ibe.service.webagent.client/src/app/models/S3Configuration.ts
export interface S3Configuration {
  serviceId: string;
  iamHost: string;
  gatewayUrl: string;
  tenantName: string;
  institutionName: string;
  timeout?: number;
  maxRetries?: number;
  collectorId: string;
  timeZone: string;
  privateKeyPath: string;
  privateKeyPassword: string;
}

File: src/Philips.IBE.Service.WebAgent/Philips.IBE.Service.WebAgent.Server/Models/S3Configuration.cs
namespace Philips.IBE.Service.WebAgent.Server.Models
{
    public class S3Configuration
    {
        public string ServiceId { get; set; } = string.Empty;
        public string TenantName { get; set; } = string.Empty;
        public string CollectorId { get; set; } = string.Empty;
        public string InstitutionName { get; set; } = string.Empty;
        public string GatewayUrl { get; set; } = string.Empty;
        public string IamHost { get; set; } = string.Empty;
        public string TimeZone { get; set; } = string.Empty;
        public string PrivateKeyPath { get; set; } = string.Empty;
        public string PrivateKeyPassword { get; set; } = string.Empty;

    }
}
